src/codegen/c.zig

```diff
@@ -1649,15 +1655,29 @@ fn airOptionalPayload(o: *Object, inst: Air.Inst.Index) !CValue {
 
 fn airStructFieldPtr(o: *Object, inst: Air.Inst.Index) !CValue {
     if (o.liveness.isUnused(inst))
-        return CValue.none;
+        return @as(CValue, CValue.none);
 
     const ty_pl = o.air.instructions.items(.data)[inst].ty_pl;
     const extra = o.air.extraData(Air.StructField, ty_pl.payload).data;
-    const writer = o.writer();
     const struct_ptr = try o.resolveInst(extra.struct_operand);
     const struct_ptr_ty = o.air.typeOf(extra.struct_operand);
+    return airStructFieldPtrFinish(o, inst, struct_ptr_ty, struct_ptr, extra.field_index);
```

Comment:
```suggestion
    return structFieldPtr(o, inst, struct_ptr_ty, struct_ptr, extra.field_index);
```

Reserve "air" prefix for the functions that are called directly from that big switch